src/renderer/views/Channel/Channel.vue

```diff
@@ -178,6 +196,29 @@
         </div>
       </div>
     </ft-card>
+    <div v-if="!isLoading && currentTab === 'community'">
+      <ft-card
+        v-for="(post, index) in latestCommunityPosts"
+        :key="index"
+        class="card"
+      >
+        <div
+          class="community-post-container"
+        >
+          <ft-community-post
+            :data="post"
+            appearance=""
+          />
+        </div>
+      </ft-card>
+      <div
+        v-if="showFetchMoreButton"
+        class="getNextPage"
+        @click="handleFetchMore"
+      >
```

Comment:
```suggestion
      <div
        v-if="showFetchMoreButton"
        class="getNextPage"
        role="button"
        tabindex="0"
        @click="handleFetchMore"
        @keydown.space.prevent="handleFetchMore"
        @keydown.enter.prevent="handleFetchMore"
      >
```

This is inaccessible.